2017/WG-09-12.md

```diff
@@ -0,0 +1,78 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the September 12th video call of WebAssembly's Working Group
+
+- **Host**: Google Hangouts
+- **Dates**: Tuesday September 14th, 2017
+- **Times**: 7:30am–8:30am Pacific Time
+- **Location**: *Brad will email Google Hangouts link to WG members prior to the meeting*
+- **Contact**:
+    - Name: Brad Nelson
+    - Email: bradnelson@google.com
+
+### Registration
+
+None required if you've attended before. Email Brad Nelson to sign up if it's
+your first time. The meeting is open to WG members only (we'll likely broaden
+this in future meetings).
+
+## Logistics
+
+The meeting will be a Google Hangouts call.
+
+## Agenda items
+
+1. Opening, welcome and roll call
+    1. Opening of the meeting
+    1. Introduction of attendees
+1. Find volunteers for note taking.
+1. Adoption of the agenda
+1. Proposals and discussions
+    1. Openness (Brad Nelson)
+       1. Discussion on how open the WG process should be.
+          * We're obviously bound by general W3C:
+          * [W3C Openness](https://www.w3.org/wiki/OpenAndTransparentW3C/Group_Openness)
+          * [Membership](https://www.w3.org/Consortium/membership-faq.html)
+       1. POLL: We will open up WG meetings to all CG members (for observation).
+       1. POLL: We will be generous in inviting non-WG members as invited experts when appropriate to WG meetings.
+    1. Mailing list
+       1. POLL: We should share the Community Group mailing lists:
+          * public-webassembly@w3.org
+          * public-webassembly-announce@w3.org
+    1. Scope of v.1 specification (Brad Nelson)
+       1. Discussion on which things should be included in the initial
+          specific, possibilities include:
+          1. [Wasm core spec](https://github.com/WebAssembly/spec)
+             1. Core format
+             1. Reference interpreter
+          1. Test suite
+          1. [JavaScript embedding](https://github.com/WebAssembly/design/blob/master/JS.md)
+          1. [Web embedding](https://github.com/WebAssembly/design/blob/master/Web.md)
+       1. POLL: We should limit v.1 to core specification only.
```

Comment:
Implicit in this question is we're leaving things out. You have a list above of what's in, could you also list what's out so that people know what we're excluding when they vote on the poll?

Reply:
done